lerna.json

```diff
@@ -0,0 +1,16 @@
+{
+  "version": "independent",
+  "npmClient": "yarn",
+  "packages": ["packages/*"],
+  "useWorkspaces": true,
+  "command": {
+    "publish": {
+      "allowBranch": "main"
```

Comment:
You have the same config as other repos except you're missing:

`"message": "Published new versions"`

Without this the commit message will just be "Publish". I would have thought for CLI you would at least want to update it to use the semantic stuff i.e. chore: Publish?

Reply:
yeah - was gonna look into that as part of the release workflow

Reply:
added a basic release workflow 👍🏻